src/app/service/constants/discordServerIds.ts

```diff
@@ -2,4 +2,6 @@ export default Object.freeze({
 	banklessDAO: '834499078434979890',
 	discordBotGarage: '851552281249972254',
 	slinkyPotatoServer: '850840267082563596',
+	bountyBoardBotServer: '905250069463326740',
```

Comment:
Same as below. There are hardcoded because they're Bankless-related servers. For the POAP distribution, we have a separate way of adding services to the list instead of hardcoding them. This file would have become unwieldy long ago if we hardcoded server IDs during the POAP rollout.